tools/postprocess.py

```diff
@@ -125,7 +125,7 @@ def create_button(
         elif group == "end-group":
                 return html + "</div>"
         else:
-            return html
+            return "<div class=\"tp-nav-button-single\">" + html + "</div>"
```

Comment:
This I think (I mean the whole layout) should be fixed in raw CSS as before.
May be I'll change that.